src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -89,9 +89,6 @@ class LSFJobManager extends AbstractLSFJobManager {
     }
 
     private Map<BEJobID, String[]> runBjobs(List<BEJobID> jobIDs) {
-        if (!executionService.isAvailable())
-            return
```

Comment:
What throws the NPE? The returned value or the executionService check?

I don't think it is a good idea to remove a check. Would it be sufficient to return an empty map instead?